lib/counter.js

```diff
@@ -25,6 +25,17 @@ function removeComments(str) {
     return str;
 }
 
+/**
+ * Empty the string literals. Do lazy match. Avoids removing
+ * everything after '//' in a string literal.
+ */
+function emptyStringLiterals(str) {
+    str = str || "";
+    str = str.replace(/\"[^\n\r]*?\"/g, "''");
```

Comment:
Ok, I guess I don't know enough about RegExp, to comment. You could ignore this PR. I only looked at the case were // is in string (ex, url in a string).